frontend/wallet/src/render/components/TransactionsView.re

```diff
@@ -11,7 +11,8 @@ module Styles = {
       gridTemplateColumns([`rem(16.), `fr(1.), `px(200)]),
       gridGap(Theme.Spacing.defaultSpacing),
       alignItems(`flexStart),
-      padding(`rem(1.)),
+      marginLeft(`rem(0.25)),
+      padding2(~h=`rem(0.75), ~v=`rem(1.)),
```

Comment:
Thought we always want these items to have 16px padding. Why do we want the spacing to be uneven?

Reply:
I think it's an optical centering deal here -- looks good in the specs. We can discuss in person, though